pallets/utility/src/lib.rs

```diff
@@ -156,5 +170,60 @@ decl_module! {
 
             Ok(())
         }
+
+        /// Relay a call as a target from an origin
+        ///
+        /// Relaying in this context refers to the ability of origin to make a call on behalf of
+        /// target.
+        ///
+        /// Fees are charged to origin
+        ///
+        /// # Parameters
+        /// - `target`: Account ID to ne relayed
+        /// - `signature`: Signature from target authorizing the relay
+        /// - `call`: Call to be relayed on behalf of target
+        ///
+        /// # Weight
+        /// - The weight of the call being relayed plus a static 250_000.
+        #[weight = (
+            call.call.get_dispatch_info().weight.saturating_add(250_000),
+            call.call.get_dispatch_info().class,
+        )]
+        pub fn relay_tx(
+            origin,
+            target: T::AccountId,
+            signature: <T as IdentityModule>::OffChainSignature,
+            call: UniqueCall<<T as Trait>::Call>
+        ) -> DispatchResult {
+            let origin = ensure_signed(origin)?;
+
+           let target_nonce = <Nonces<T>>::get(&target);
+
+            ensure!(
+                target_nonce == call.nonce,
+                Error::<T>::InvalidNonce
+            );
+
+            ensure!(
+                signature.verify(call.encode().as_slice(), &target),
+                Error::<T>::InvalidSignature
+            );
+
+            ensure!(
```

Comment:
isn't get automatically checked in signed extension?. So we don't need this

Reply:
Yeas, this check can be removed @CJP10